CHANGELOG.md

```diff
@@ -10,8 +10,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Fixed
 
-- Authentication renater select crashing the site on scroll
```

Comment:
In your commit description

`s/Check if local storage is available vore using it/Check if local storage is available before using it`